spec/models/appeal_spec.rb

```diff
@@ -998,6 +998,26 @@
     end
   end
 
+  describe ".cavc_remand" do
```

Comment:
I believe the convention is to use '#' for instance methods:
```suggestion
  describe "#cavc_remand" do
```